content/manuals/docker-hub/image-library/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Learn about Docker Hub's library of images, extensions, and plugins.
+keywords: Docker Hub, Hub, content library
+title: Container image library
+linkTitle: Library
+weight: 20
+---
+
+Docker Hub's container image library is the world's largest collection of
```

Comment:
question: How about content library rather than container library, since it contains not only images but also plugins, extensions, and oci artifacts. (Also: "trusted content")

```suggestion
title: Content library
linkTitle: Library
weight: 20
---

Docker Hub's content library is the world's largest collection of
```